Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -1944,6 +1945,24 @@ def run(self):
         return super().run()
 
 
+class CleanSaferCPPArchive(shell.ShellCommandNewStyle):
+    name = 'clean-safer-cpp-archive'
+    description = ['cleaning safer cpp archive']
+    descriptionDone = ['cleaned safer cpp archive']
+    haltOnFailure = False
+    flunkOnFailure = False
+    warnOnFailure = False
+
+    @defer.inlineCallbacks
+    def run(self):
+        self.log_observer = logobserver.BufferLogObserver()
+        self.addLogObserver('stdio', self.log_observer)
+        archive_dir = f"{self.getProperty('builddir')}/smart-pointer-result-archive/{self.getProperty('buildnumber')}"
```

Comment:
Since this directory name `mart-pointer-result-archive` is hard-coded in multiple places, it might be a good idea to define it as a variable (near top of this file) and re-use it. Although that can be done in a separate PR.